cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/util/ConfigUtils.java

```diff
@@ -25,4 +26,14 @@ public static ApiConfig makeApiConfig(String name, String value) {
         apiConfig.setValue(value);
         return apiConfig;
     }
+
+    public static void addConfig(ApiRole apiRole, ApiConfig apiConfig) {
```

Comment:
I think it could be part of CmConfigServiceDelegate interface as a default method.